File: api/v2010/call/create-voice-example/create-voice-example.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

call = @client.calls
  .create(
     url: 'http://twimlets.com/message?Message%5B0%5D=Hello%20from%20your%20subaccount',
     to: '+16518675310',
     from: '+14158141829'
   )

puts call.sid

File: api/v2010/call/create-voice-example/create-voice-example.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var call = CallResource.Create(
            url: new Uri("http://twimlets.com/message?Message%5B0%5D=Hello%20from%20your%20subaccount"),
            to: new Twilio.Types.PhoneNumber("+16518675310"),
            from: new Twilio.Types.PhoneNumber("+14158141829")
        );

        Console.WriteLine(call.Sid);
    }
}
